### PR TITLE
Creation of MTBseq lablog_results

### LINKED ISSUE #232: Creation of MTBseq lablog_results
Creation of a file that automates the creation of symbolic links for the main results files in the results directory.

### PR DESCRIPTION
**Pull request by jaimeozaez**
Creation of a file that automates the creation of symbolic links for the main results files in the results directory.

Closes #232 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] Make sure your code lints (`black and flake8`).
- If a new tamplate was added make sure:
    - [ ] Template's schema is added in `templates/services.json`.
    - [ ] Template's pipeline's documentation in `assets/reports/md/template.md` is added.
    - [ ] Results Documentation in `assets/reports/results/template.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).
- [ ] If you know a new user was added to the SFTP, make sure you added it to `templates/sftp_user.json`

### CODE REVIEW

File: bu_isciii/templates/mtbseq/RESULTS/lablog_mtbseq_results [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+cd $DELIVERY_FOLDER/mtbseq
+
+ln -s ../../../ANALYSIS/*MTBSEQ/03-MTBSeq/all_samples/Amend .
+ln -s ../../../ANALYSIS/*MTBSEQ/03-MTBSeq/all_samples/Groups .
+ln -s ../../../ANALYSIS/*MTBSEQ/03-MTBSeq/resistances_all/ .
+ln -s ../../../ANALYSIS/*MTBSEQ/03-MTBSeq/stats_all/statistics_all.tab .
+ln -s ../../../ANALYSIS/*MTBSEQ/03-MTBSeq/classification_all/strain_classification_all.tab .
+
+find . -xtype l -delete
```
Comment (saramonzon) [inline bu_isciii/templates/mtbseq/RESULTS/lablog_mtbseq_results:13]: Doesn't this delete all the links you've just created?

Comment (jaimeozaez) [reply bu_isciii/templates/mtbseq/RESULTS/lablog_mtbseq_results:13]: Just the broken ones

Comment (jaimeozaez) [reply bu_isciii/templates/mtbseq/RESULTS/lablog_mtbseq_results:13]: This line deletes the links from folder "all_samples", that most of times will be empty

Comment (saramonzon) [reply bu_isciii/templates/mtbseq/RESULTS/lablog_mtbseq_results:13]: ah ok! Perfect :)

**Review by saramonzon — APPROVED** [review]

**Pull request merged**